solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -16,7 +16,6 @@
  */
```

Comment:
This PR doesn't change the failures of TestBulkSchemaConcurrent. The first error I see in a filed run is an NPE at
at org.apache.solr.core.SolrCore.close(SolrCore.java:1573)

      coreMetricManager.close();

which are _not_ thrown for passing iterations. I backed out to just before SOLR-14151 and don't have this problem, whereas as soon as I went from that commit, this fails beasting for me very quickly.

So while this patch seems a reasonable place to start, it doesn't address the failures.